src/components/breadcrumbs/breadcrumbs.types.ts

```diff
@@ -0,0 +1,28 @@
+import { Link } from '@limetech/lime-elements';
+
+export interface BreadcrumbItem {
+    /**
+     * label displayed on the step.
+     */
+    text: string;
+
+    /**
+     * Icon of the step.
+     */
+    icon?: string;
+
+    /**
+     * If `true`, text will be rendered as a tooltip
+     */
+    iconOnly?: boolean;
```

Comment:
I think I mentioned this in another PR, but will mention it here as well. It feels wrong to name a property like this, because it might become mutually exclusive with other props that gets introduced later. What would happen if all these are set? 

```ts
{
    iconOnly: true,
    textOnly: true,
    linkOnly: true,
}
```

It becomes hard to know what behavior this will give us since these props will not work together at the same time. This might not be a realistic example for this component, but I hope you get my point anyway :smile: 

A better alternative could be to either have a `type` property on the item that would give an indication on how it should be used/rendered, or have a property named `disableText` or similar

Reply:
🤔 it makes sense. But I don't remember why we didn't change the `type` after [the discussion](https://github.com/Lundalogik/lime-elements/pull/2315#discussion_r1214000403) for Action Bar. Why did we let the other one in the Actionbar be merged with this name? 

Is it better to keep it consistent? Or are you suggesting to rename both?

In that case, I'd prefer a type like `hideText`, but that doesn't look nice neither? Because one could argue, what about `hideIcon` and `hideLink` and so forth. Or? In this case, a better name could be `renderTextAsTooltip` or `textAsTooltip` maybe?

Anyhow, not sure what to do here.... Continue, or change this in both components?

Reply:
Oh I see now what you mean with `type`. Yes that's perhaps a better way. But we have two types right now. It's either `iconAndText` (the default) or `iconOnly` (which will render the text in the tooltip). So it's possible to change that.

> We can't change the name in the action bar now without making a breaking change, but we can add an additional property with the new name and deprecate the old one and then remove it later.

Nobody is using the component yet except us. 
https://github.com/search?q=org%3ALundalogik%20limel-action-bar&type=code

So I don't think we should be bureaucratic about this at all, and change it in Elements and CRM components in a quick PR now. It's fine if it's a BREAKING CHANGE. Since nobody is using it anyways. Please let's not deprecate it. We just make it a breaking change.

Green light?

Reply:
> Oh I see now what you mean with type. Yes that's perhaps a better way. But we have two types right now. It's either iconAndText (the default) or iconOnly (which will render the text in the tooltip). So it's possible to change that.

I don't think you should be forced to set a type. We can include `iconAndText` if you want, but I think it should default to that anyway if nothing has been set

> So I don't think we should be bureaucratic about this at all, and change it in Elements and CRM components in a quick PR now. It's fine if it's a BREAKING CHANGE. Since nobody is using it anyways. Please let's not deprecate it. We just make it a breaking change.

This is why we should have stuff in beta for a while before releasing it :smile:

Reply:
> This is why we should have stuff in beta for a while before releasing it 😄

You're technically right 😄 But practically not 🤣 because nobody is using Lime Elements, except Lime CRM folks 🤪

Reply:
This is still an issue

Reply:
Ok, I've now added a `type`. So questions again:
1. What convention should we use for types? `icon-only` or `iconOnly`?
2. Can we decide a name? I don't find a good name.